config/bidderinfo.go

```diff
@@ -22,6 +22,11 @@ type BidderInfo struct {
 	GVLVendorID             uint16               `yaml:"gvlVendorID"`
 	Syncer                  *Syncer              `yaml:"userSync"`
 	Experiment              BidderInfoExperiment `yaml:"experiment"`
+	CompressionType         *CompressionType     `yaml:"compressionType"`
+}
+
+type CompressionType struct {
+	EndpointCompression string `yaml:"endpointCompression"`
```

Comment:
Would it be better to be more specific here so that it is understood that this compression is for outgoing requests only?

There is already confusion around how Go handles GZIP in it's default httpclient.

My thoughts are that if we're more specific here the config will be better understood and easier to reason with.

Something like `RequestCompressionType`? Or something along those lines?

Reply:
I understand your reasoning, but we are trying to be consistent with variable/object naming between PBS-Go and PBS-Java, so I think to accomplish this this object can stay being named this way. I'll see if other team members think differently though!